instrumentation/netty/netty-4.1/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/netty/v4_1/client/NettyClientSingletons.java

```diff
@@ -24,21 +24,21 @@
       Config.get().getBoolean("otel.instrumentation.netty.always-create-connect-span", false);
```

Comment:
Can we still change property names? According to VERSIONING.md

> Changes to configuration properties (...) will be considered breaking changes (unless they only affect telemetry produced by instrumentation)

This only affects generated telemetry, so maybe it'd be okay to rename it? I've been thinking about renaming this to sth like `otel.instrumentation.<module>.connection-telemetry.enabled`

Reply:
Yes we can rename since it only affects telemetry shape

Reply:
Nice. I'll do that in a separate PR, together with docs for the new property.